src/resources/i18n/translations/en.json

```
@@ -260,20 +260,20 @@
   "Custom API domain": "Custom API domain",
   "Custom Feeds": "Custom Feeds",
   "Custom RSS feeds to parse in parallel": "Custom RSS feeds to parse in parallel",
-  "Custom RSS feeds to parse in parallel helper text": "Parsing too many feeds in parallel may cause issues on some devices. 3 is recommended.",
+  "Custom RSS feeds to parse in parallel helper text": "Parsing more than three feeds at once may cause issues on some devices.",
   "Custom web domain": "Web domain",
-  "Custom web domain subtext": "Custom domains to use an alternate server for your app data.",
+  "Custom web domain subtext": "These allow choice of app data server.",
   "Dark Mode": "Dark",
   "Data Settings": "Data Settings",
-  "Default downloaded episode limit for each podcast": "Max downloaded episodes for each podcast",
+  "Default downloaded episode limit for each podcast": "Limit",
```

```suggestion
  "Default downloaded episode limit for each podcast": "Individual episode-limits",
```
```suggestion
  "Limit the number of downloaded episodes for each podcast by default": "Number of episodes to keep.",
```
```suggestion
  "Limit the number of downloaded episodes for each podcast by default": "Limit kept episodes to",
```
in any event with just the number below, so no "Limit" string.

---

I like dropping the "Resume:" and instead changing the buttons to say "Resume" and "Cancel" in the alert. I just made the change in this commit: https://github.com/podverse/podverse-rn/commit/abef2da7e9f5f963faafc7b2930986d9fb029f04

As for the "Limit", I think "Number of episodes to keep" is good. I'll accept that one.